package.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/204918886-a348f285-0d39-4b30-9cb8-761d79d0fc2a.PNG)

- [ ] Your license link is not working 😨 Kindly fix this link to make your license file easily accessible to all viewers and also to make your readme descriptive 😊